setup.cfg

```diff
@@ -61,14 +61,16 @@ oracle =
 postgresql = psycopg2>=2.7
 postgresql_pg8000 = pg8000>=1.16.6
 postgresql_asyncpg =
+    %(asyncio)s
```

Comment:
oh nice. I did not know you could to this.

is it documented somewhere?
I'm not sure if we should use it if it's not documented, since all the setup stuff is not overly stable, see pep517 that still has some problems

Reply:
it's a feature of ConfigParser: https://docs.python.org/3/library/configparser.html#interpolation-of-values

Reply:
I somehow assumed that setuptools would has this documented, but I gess it just used the builtin config parser.

It's ok for me.

Also note that asyncio is really superfluous, since at the moment the beta install `greenlet` in all cases. See line 43.

@zzzeek is the plan to keep `greenlet` as a requirement also for the stable 1.4 series?